src/main/java/com/kk/strong/model/dto/UserDto.java

```diff
@@ -0,0 +1,19 @@
+package com.kk.strong.model.dto;
+
+import com.kk.strong.model.GenderType;
+import lombok.Data;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Data
+public class UserDto {
+
+    private Long id;
+    private String name;
+    private GenderType gender;
+    private int age;
+    private int height;
+    private List<BodyReportDto> bodyReports = new ArrayList<>();
+    private List<WorkoutSessionDto> workoutSessions = new ArrayList<>();
```

Comment:
Not sure if it wouldn't be better to return lists of ids only. It's convenient to have access to all of this data when fetching a user, but maybe it should be more private; also when getting all users it could be slow

Reply:
It depends on the logic you want to do, will you always need a full list of dtos where fetching the user? Maybe instead you just need only a part of the information from this dto and you can create a smaller version of dto here and fetch less from database? Pagination also can take part in this etc

Reply:
Ok, I'll leave it as it is, but will keep that in mind